etc/visual-studio/libopenthread-windows.vcxproj

```diff
@@ -1,4 +1,4 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Any idea what changed on this line? Was it a UTF format marker prefix or something?